packages/remix-serve/index.ts

```diff
@@ -8,7 +8,26 @@ export function createApp(buildPath: string, mode = "production") {
 
   app.disable("x-powered-by");
 
-  app.use(compression());
+  app.use(
+    compression({
```

Comment:
Maybe utilitize this function (for other remix-express users that aren't using one of our templates) and possibly check res content-type to know what to compress instead of checking the req.